src/entities/schema.ts

```diff
@@ -195,7 +195,7 @@ export class Item extends Entity {
     this.set("rawMetadata", Value.fromString(""));
     this.set("isApproved", Value.fromBoolean(false));
     this.set("thirdParty", Value.fromString(""));
-    this.set("searchThirdPartyId", Value.fromString(""));
+    this.set("reviewedAt", Value.fromBigInt(BigInt.zero()));
```

Comment:
I think that removing `searchThirdPartyId` will break our code. I would suggest we keep it.